### ISSUE TITLE
启动的时候pad一直报错，这个问题怎么解决？

### ISSUE THREAD

**Issue by chenlei528884440** [modelscope/FunASR#2701]
[W shape_analysis.cpp:841] failed PropagateTensorShapeOnNode with schema: 
aten::pad(Tensor self, int[] pad, str mode="constant", float? value=None) -> (Tensor)
%input.987 : Float(*, *, *, requires_grad=0, device=cuda:0) = aten::pad(%input.985, %5760, %5761, %5762), scope: __module.model.encoders/__module.model.encoders.48/__module.model.encoders.48.self_attn/__module.model.encoders.48.self_attn.pad_fn # /usr/local/lib/python3.8/dist-packages/torch/nn/modules/padding.py:23:0

**Comment by LauraGPT** [comment]
Closing due to inactivity (7+ months). Please reopen if still relevant with latest version: `pip install funasr>=1.3.3`

Docs: https://modelscope.github.io/FunASR/tutorial.html
